terraform/monitoring/config/alert.test.yml

```diff
@@ -28,6 +28,8 @@ tests:
           - series: 'requests{app="teaching-vacancies-production",guid="c95fd761-9b05-4527-829b-fd691a6c755a",instance="1",organisation="dfe",space="teaching-vacancies-production",status_range="5xx"}'
             values: '0+0x4 0+0x4 0+0x4 0+0x4 0+0x4 0+0x4 0+0x4'
 
+      name: Prod_Requests_Failures_Elevated
```

Comment:
If this is meant tim be a title, could it be moved to the top of the section to increase awareness?
Also it may be less confusing if it has the name of either the exact alert name (`ProdRequestsFailuresElevated`) or a human readable title (`Test ProdRequestsFailuresElevated alert`)

Reply:
done